src/components/cards/controltable/ControlRowCol.vue

```diff
@@ -2,9 +2,17 @@
   <v-row>
     <v-col cols="12" sm="12" md="12" lg="1" xl="1">
       <v-layout class="pl-2" fill-height justify-center align-center>
-        <v-btn :color="status_color" block depressed>
-          <h3>{{ result.status.toUpperCase() }}</h3>
-        </v-btn>
+        <table width="100%">
+          <tr>
+            <td style="text-align:center">
+              <v-card :color="status_color">
+                <v-card-text class="pa-2 font-weight-bold">
+                  <h3>{{ result.status.toUpperCase() }}</h3>
+                </v-card-text>
+              </v-card>
+            </td>
+          </tr>
+        </table>
```

Comment:
I think adding `class='disabled-btn'` and adding a `disabled-btn` class in the css below as described in https://stackoverflow.com/a/54309684/1670307 is probably a simpler solution, add an inline table for this seems a little overkill.

Reply:
Created a new classname called 'unclickablebutton' as a class attribute on the button element.  Created a scoped CSS entry on the page to disable the button and remove transparency.

The following files were modified:
+ /src/components/cards/controltable/ControlRowCol.vue

Reply:
Removed the transparency attribute from the CSS section.